string-handling/string-characteristics/python/case_conversion.py

```diff
@@ -0,0 +1,20 @@
+# Isaac Computer Science
+# Usage licensed under the Open Government Licence v3.0
+
+def convert_to_lower(letter):
+    """Demonstrates how to use ASCII codes to convert letter case"""
+    letter_code = ord(letter)
```

Comment:
This is inconsistent with the C# code because the python version assumes letter is a single character. I suggest changing the C# one to do that too

Reply:
done